src/bp-messages/classes/class-bp-messages-rest-controller.php

```diff
@@ -1548,7 +1549,7 @@ public function get_collection_params() {
 		);
 
 		$params['type'] = array(
-			'description'       => __( 'Filter the result by thread status.', 'buddypress' ),
+			'description'       => __( 'Filter threads by its status.', 'buddypress' ),
```

Comment:
```suggestion
			'description'       => __( 'Filter threads by their status.', 'buddypress' ),
```

Or just "Filter threads by the status".